wdl2cwl/main.py

```diff
@@ -419,6 +416,31 @@ def get_memory_literal(self, memory_runtime: WDL.Expr.String) -> float:
 
         return memory
 
+    def get_outdir_requirement(
+        self, outdir: Union[WDL.Expr.Get, WDL.Expr.Apply]
+    ) -> int:
+        """Produce the memory requirement for the output directory from WDL runtime disks."""
+        with WDLSourceLine(outdir, ConversionException):
+            if isinstance(outdir, (WDL.Expr.Apply, WDL.Expr.String)):
+                list_object = (
+                    outdir.arguments if hasattr(outdir, "arguments") else outdir.parts  # type: ignore
```

Comment:
If outdir is an expression WDL.Expr.String then it has an iterable object called parts else if it's a WDL.Expr.Apply it has an iterable object called arguments.